utils/lists.py

```diff
@@ -6,36 +6,28 @@
     "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
     'senpai, pls no ;-;'
 ]
+i = 0
+answers = []
+questions = []
+
+
+with open ("trivia_questions_answers.txt") as fp:
+    count = 0
+    a_count = 0
+    q_count = 0
+    
+    for line in fp:
+        count +=1
+        if count % 2 == 0: # answer
+            answers.append(line)
+            a_count+=1
+        else: # question
+            questions.append(line)
+            q_count+=1
```

Comment:
This could all be put into a function. It would make the code more readable but it doesn't have to be put into a function.